src/tools/rustfmt/src/parse/macros/mod.rs

```diff
@@ -98,7 +98,7 @@ fn check_keyword<'a, 'b: 'a>(parser: &'a mut Parser<'b>) -> Option<MacroArg> {
     None
 }
 
-pub(crate) fn parse_macro_args(
+pub fn parse_macro_args(
```

Comment:
Is the `pub(crate)` visibility within rustfmt trigger any lints? If not, could we keep this as `pub(crate)` for now?

Reply:
Good catch, that change is unnecessary. I have reverted it.

Reply:
Thank you!